Project/Source/Modules/ModulePrograms.h

```diff
@@ -51,5 +51,5 @@ class ModulePrograms : public Module {
 
 	// Particle Shaders
 	ProgramBillboard* billboard = 0;
-	unsigned trail = 0;
+	ProgramTrail* trail = 0;
```

Comment:
Jaime did a PR changing the 0 to nullptr for all the programs.
To keep consistence remember and try to modify it also, now or when merging.